cg/apps/crunchy.py

```diff
@@ -0,0 +1,268 @@
+"""
+    Module for compressing bam to cram
+"""
+
+import logging
+import tempfile
+from pathlib import Path
+
+from cg.utils import Process
+from .constants import (
+    BAM_SUFFIX,
+    CRAM_SUFFIX,
+    CRAM_INDEX_SUFFIX,
+    FASTQ_FIRST_SUFFIX,
+    FASTQ_SECOND_SUFFIX,
+    SPRING_SUFFIX,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+SBATCH_HEADER_TEMPLATE = """
+#!/bin/bash
+#SBATCH -J {job_name}
+#SBATCH -A {account}
+#SBATCH -n 1
+$SBATCH --stderr {log_dir}/{job_name}.stderr
+$SBATCH --stdout {log_dir}/{job_name}.stdout
+
+source activate {crunchy_env}
+"""
+
+SBATCH_BAM_TO_CRAM = """
+crunchy compress bam --bam-path {bam_path} --cram-path {cram_path} --reference {reference_path}
```

Comment:
```suggestion
crunchy --reference {reference_path} compress bam --bam-path {bam_path} --cram-path {cram_path}
```
as I have implemented it we need to point to reference in base command